docs/spec/manifest-v2-2.md

```diff
@@ -0,0 +1,242 @@
+        If the length of the downloaded content does not match the specified
+        length, the content should not be trusted.
+    
+    - **`digest`** *string*
+    
+        A string labeled with a digest algorithm prefix and the hex-encoded
+        value of that digest for the content.
+
+        Digest strings should be of the form:
+
+            <digest-alg-name>:<hex-encoded-digest>
+
+        A `digest` in this form is considered relative to the repository which
+        this manifest also comes from. The `<digest-alg-name>` should be alpha-
+        numeric and may contain the characters `+`, `-`, `_`, and `.`. Example
+        `<digest-alg-name>`s are:
+
+            sha256
+            sha512
+            sha3_256
+            sha3_512
+
+        The `<hex-encoded-digest>` value must contain hexadecimal digits only
+        and should be long enough to encode the full digest produced by the
+        corresponding digest algorithm.
+
+        Optionally, the `digest` value may be an absolute reference to a digest
+        in another repository:
+
+            <repository>@<digest-alg-name>:<hex-encoded-digest>
+
+        A `digest` in this form is considered relative to the repository
+        explicitly specified in the `<repository>` section of the reference.
+        The `<repository>` value must match the format specified by the
+        [Namespace and Discovery Documentation](https://github.com/dmcgowan/distribution/blob/namespace-proposal/doc/spec/namespace.md#name-grammar).
+    
+    Any media type may be in the `mediaType` field as the application may
+    use any type of content. If the `mediaType` is
+    `application/vnd.docker.distribution.content.manifest.v2+json` then the
+    dependency is another content manifest as described by this document. The
+    dependencies from that manifest should also be considered dependencies of
+    this application.
+
+    In general, the ordering of these dependencies is significant as it may be
+    significant to the application. However, it is best for the application
+    data below to also designate what actions to perform on the dependencies.
+
+- **`application`** *object*
+
+    Fields are:
+    
+    - **`mediaType`** *string*
+    
+        The MIME type of the JSON object in the `data` field.
+    
+    - **`data`** *object*
+    
+        Abstract application data in any schema, as described by the
+        `mediaType` field.
+    
+    Application data may be *any* JSON object, allowing this manifest format,
+    along with the generic dependencies list, to support any type of 
+    application that can be represented as a JSON configuration and a
+    collection of content-addressable blobs of data.
+
+- **`meta`** *object*
+
+    Meta may be *any* JSON object, allowing content creators to annotate
+    their application manifest with any additional labels beyond those already
+    defined by this manifest format. 
+
+### Example Manifest
+
+```json
```

Comment:
@thaJeztah Project rule: if it works on github, it should work on the docs site. We cannot push limitations in our documentation system onto contributors. Github Flavored Markdown is a thing and there is no reason not to have support for it in the documentation system. Put another way, if it renders on github but not the docs site, file a bug for the docs site. We cannot reliably police the differences there is no good excuse to not support this syntax.